data/tools/wmllint

```diff
@@ -2503,7 +2504,7 @@ to be called on their own".format(filename, num))
                 if capitalization_error.search(lines[i]):
                     print('"%s", line %d: probable capitalization or punctuation error' \
                           % (filename, i+1))
-                if key == "message" and in_message and not in_option and not ("wmllint: ignore" in comment or "wmllint: noconvert" in comment):
+                if False:
```

Comment:
Is this something that should be here, or a left-over from debugging?

Reply:
Oops